app/helpers/users_helper.rb

```diff
@@ -6,9 +6,9 @@ def gravatar_for user, size: Settings.helper.users.img_size
   end
 
   def display_err errors, field
-    if errors.present?
-      err = "<div class=\"invalid-feedback\"> #{errors.full_messages_for(field)[0]}</div>" if errors.include?(field)
-      err.html_safe if err.present?
-    end
+    return if errors.blank?
+
+    err = errors.full_messages_for(field)[0] if errors.include?(field)
+    content_tag(:div, err, class: "invalid-feedback") if err.present?
```

Comment:
Tại sao cần viết như thế này?

Reply:
phần này, em hiển thị lỗi đối với từng form-control của form signup, nên em viết helper để tránh lặp lại code
+ errors lưu lại toàn bộ lỗi khi form được submit rồi validate, có thể nil, mảng rỗng, nên line 9 check có lỗi text không ạ.
+ line 11 sẽ lấy ra lỗi validate đầu tiên của trường field, ví dụ email có validate length, format thì nó sẽ lấy lỗi của 1 validate duy nhất. 
+ content_tag sẽ trả về khối HTML với nội dung là nội dung của lỗi validate phía trên.
![Screenshot from 2020-08-11 15-03-44](https://user-images.githubusercontent.com/68833449/89872937-dffa3f80-dbe3-11ea-80ac-723103c1cacf.png)